sbt/plugin/src/main/scala/ch/epfl/scala/debugadapter/sbtplugin/DebugAdapterPlugin.scala

```diff
@@ -325,6 +390,38 @@ object DebugAdapterPlugin extends sbt.AutoPlugin {
       }
     }
 
+  /**
+   * Get fork options for teh given test group taking into account
+   * additional jvm options and env variables.
+   */
+  private def getForkOptions(
+      testGroup: Group,
+      defaultForkOpts: ForkOptions,
+      jvmOptions: Vector[String],
+      environmentVariables: Vector[String]
+  ) = {
+    val forkOptions = testGroup.runPolicy match {
+      case InProcess => defaultForkOpts
+      case SubProcess(forkOpts) => forkOpts
+    }
+
+    val additionalEnv = environmentVariables
+      .foldLeft(Map.empty[String, String]) { case (env, line) =>
+        val idx = line.indexOf("=")
+        if (idx > 0 && idx != line.length - 1) {
+          val key = line.substring(0, idx)
+          val value = line.substring(idx + 1)
+          env + (key -> value)
+        } else env
```

Comment:
Maybe this would be slightly easier to read:

```suggestion
        line.split('=') match {
           case Array(key, value) => env + (key -> value)
           case _ => env
         }
`````

Reply:
Seems a lot nicer! I copied this code from Bloop and I didn't think about it enough